sql-select/five-customer-emails.sql

```diff
@@ -0,0 +1,3 @@
+select  "email"
+  from  "customers"
+  limit 5;
```

Comment:
You're missing some fields:

---

![image](https://user-images.githubusercontent.com/16143771/179878691-43041597-abff-49bf-9383-25d574af9fe0.png)

Reply:
![image](https://user-images.githubusercontent.com/97073517/179881927-dbe1f0c5-125e-472e-b5de-8ccb2555ae1e.png)